assets/i18n/zh-tw.json

```
@@ -0,0 +1,269 @@
+    "dgc-title": "狗狗幣",
+    "dgc-title-firstcap": "狗狗幣", 
+    "dgc-title-caps": "狗狗幣", 
+    "dgc-alt-watch-video": "看影片", 
+    "dgc-alt-logo": "狗狗幣 圖標", 
+    "dgc-header-desc": "狗狗幣是一種開源的加密貨幣，全世界的狗狗都很喜歡。", 
+    "dgc-start-using": "開始使用狗狗幣吧：",
+    "dgc-wallet-windows": " Windows", 
+    "dgc-wallet-osx": " macOS", 
+    "dgc-wallet-linux": " Linux", 
+    "dgc-wallet-ios": " iOS", 
+    "dgc-wallet-android": " Android", 
+    "dgc-wallet-blackberry": " Blackberry", 
+    "dgc-wallet-browser": " 網頁版", 
+    "dgc-wallet-source": " 原始碼", 
+    "dgc-scroll-button-link": "點擊後向下滾動", 
+    "dgc-scroll-button-alt": "向下滾動按鈕", 
+    "dgc-dogepng-alt": "狗", 
+    "dgc-what-is-doge": "什麼是狗狗幣？", 
+    "dgc-quick-desc": "狗狗幣是去中心化加密貨幣，讓您能輕鬆的在網路上傳送貨幣。簡而言之就是 \"網路貨幣\"",
+    "dgc-get-started-now": "開始使用狗狗幣", 
+    "dgc-learn-more": "了解更多", 
+    "dgc-what-with-dog": "什麼是狗狗幣 它是狗嗎？", 
+    "dgc-doge-explained": "\"狗\" 只是個有趣的命名, 就像是吉祥物！作為流行迷因的日本柴犬就永遠鑲嵌進狗狗幣中了！", 
+    "dgc-doge-learn": "了解更多關於狗狗幣的事", 
+    "dgc-shiba-inu": "柴犬", 
+    "dgc-fun-currency": "狗狗幣是有趣也很友好的加密貨幣。", 
+    "dgc-set-apart": "狗狗幣之所以與眾不同、脫穎而出，全因它在網路上有強大的流通性！",
+    "dgc-reddit": "Reddit 社群", 
+    "dgc-dogeducation": "Dogeducation Reddit 社群",
+    "dgc-twitter": "Twitter",
+    "dgc-discord": "Discord 社群",
+    "dgc-getting-started": "如何開始認識有關狗狗幣的一切", 
+    "dgc-setup-wallet": "設定狗狗幣錢包非常 <strong>簡單</strong> 只需 1, 2, 3 步驟. 認真說！ 查看以下教學流程", 
+    "dgc-edu-help": "如果您需要幫忙, 請來到 r/dogeducation on Reddit! <br /> 樂於助人的我們將回答您的任何問題！",
+    "dgc-footer-waiting": "<strong>還在等什麼？</strong> 現在就開始使用狗狗幣吧！", 
+    "dgc-footer-getting-started": "點擊此使用", 
+    "dgc-footer-learn-how": "學習如何使用狗狗幣", 
+    "dgc-community-title": "社群.", 
+    "dgc-community-participate": "參與狗狗幣社群", 
+    "dgc-get-dogecoin-title": "獲得狗狗幣", 
+    "dgc-get-dogecoin-how": "了解如何獲得狗狗幣",
+    "dgc-stores-title": "商店", 
+    "dgc-stores-buy": "用狗狗幣支付",
+    "dgc-footer-credits": "Created by <a href=\"https://ummjackson.com\" target=\"_blank\">Jackson Palmer</a> & <a href=\"https://twitter.com/BillyM2k\" target=\"_blank\">Shibetoshi Nakamoto</a>. Website Design by <a href=\"https://www.reddit.com/user/edswf1\" target=\"_blank\">/u/edswf1</a>. Logo by <a href=\"http://christinemix.com\" target=\"_blank\">Christine Ricks</a>.",
+    "dgc-footer-copyright": "© The Ðogecoin Project", 
+    "dgc-browser-modal-title": "選擇網路錢包：", 
+    "dgc-browser-modal-online": "網路錢包是使用狗狗幣最快速且簡單的方法，但安全性會不如在您的離線電腦上所保存的錢包。 <br /><br />在任何時候，我們建議您的線上錢包中只存放少許的狗狗幣。 <br />這裡有許多基於瀏覽器的錢包:",
+    "dgc-browser-modal-dogechain": "DogeChain Wallet", 
+    "dgc-browser-modal-holytransaction": "HolyTransaction", 
+    "dgc-browser-modal-blockio": "Block.io", 
+    "dgc-browser-modal-online-disclaimer": "<strong>注意：</strong> 狗狗幣不負責與維護這些線上錢包。 儘管我們相信背後的組織具有可信度，但請自行斟酌風險。", 
+    "dgc-ios-wallet-modal-important": "重要：", 
+    "dgc-ios-wallet-modal-view-only": "iOS 上的 MyDoge 僅能用來查看錢包。", 
+    "dgc-ios-wallet-modal-not-send": "MyDoge <strong>不能</strong> 發送狗狗幣. 所以您必须自行建立實際的狗狗幣錢包.", 
```

Original sentence "MyDoge <strong>cannot</strong> send Dogecoins. You must set up an actual Dogecoin wallet before you can use this Dogecoin Wallet viewer."
```suggestion
    "dgc-ios-wallet-modal-not-send": "MyDoge <strong>不能</strong> 發送狗狗幣，在使用 MyDoge 前必須先建立一個真正的錢包。", 
```